pkgs/top-level/aliases.nix

```
@@ -895,6 +895,7 @@ mapAliases ({
   rr-unstable = rr; # Added 2022-09-17
   rtl8723bs-firmware = throw "rtl8723bs-firmware was added in mainline kernel version 4.12"; # Added 2023-07-03
   rtsp-simple-server = throw "rtsp-simple-server is rebranded as mediamtx, including default config path update"; # Added 2023-04-11
+  rtx = throw "rtx was rebranded to mise"; # Added 2024-01-05
```

Is this drop in? see https://github.com/jdx/mise#migrating-from-rtx

```suggestion
  rtx = mise; # Added 2024-01-05
```

---

updated